gr-uhd/lib/rfnoc_tx_streamer_impl.h

```diff
@@ -32,6 +32,9 @@ class rfnoc_tx_streamer_impl : public rfnoc_tx_streamer
              gr_vector_const_void_star& input_items,
              gr_vector_void_star& output_items) override;
 
+    /***** Message Handlers *************************************************/
+    void handle_pdu(pmt::pmt_t msg);
```

Comment:
```suggestion
    void handle_pdu(const pmt::pmt_t& msg);
```
to avoid the unnecessary refcount increase/decrease of the shared_ptr, which is mutexed